Packs/CortexXDR/ReleaseNotes/2_9_2.md

```diff
@@ -1,5 +1,6 @@
-
 #### Integrations
```

Comment:
@yaakovpraisler you are editing an existing RN! Please revert this change and add a new file and update the pack version